clang/lib/DPCT/MigrationReport/Statics.cpp

```diff
@@ -319,6 +323,20 @@ class AnalysisModeStats {
       Total += Summary;
     }
     Total.dump(OS, Indent);
+    if (DependencyBits.Flag) {
+      LineStream(OS, Indent)
+          << llvm::raw_ostream::Colors::BLUE
+          << "Library Dependencies of SYCL Project:" << llvm::raw_ostream::Colors::RESET;
+      static std::string DependenceNames[] = {
+          "oneAPI Math Kernel Library (oneMKL)", "oneAPI DPC++ Library(oneDPL)",
```

Comment:
Could you try to list them in *inc file, then maybe easy for review and extent for new lib like oneSHMEM and so on.